### DISCUSSION TITLE
pretix.cfg in development installation

### DISCUSSION THREAD

**Discussion by giorgioUnimi** [pretix/pretix#4531] [Q&A]
Hi to all, I'm new to pretix. I have followed these instrctions https://docs.pretix.eu/en/latest/development/setup.html and the installation ha  gone well but inside the guide it talks about the file src/pretix.cfg which I cant find in my pretix folder. Why ?

**Comment by antropolouco** [comment]
Hey there!

I'm new here too, but I can answer that.

You may create your own pretix.cfg text file under the pretix/src directory.

Check [https://docs.pretix.eu/en/latest/admin/config.html#pretix-settings](https://docs.pretix.eu/en/latest/admin/config.html#pretix-settings) for further information.

Here is my working pretix.cfg file example:
```

[pretix]
instance_name=myinstancename
url=https://myurl.com
currency=WHATEVER
datadir=/data
plugins_default=pretix.plugins.sendmail,pretix.plugins.statistics
trust_x_forwarded_for=on
trust_x_forwarded_proto=on
trust_x_forwarded_host=on

[mail]
from=your@email
custom_sender_verification_required=True
custom_sender_spf_string=True
custom_smtp_allow_private_networks=True
host=smtp.server.com
port=587
user=youruser
password=yourpass
tls=True
```

Best regards
